### DISCUSSION TITLE
Connessione HTTPS non funziona più

### DISCUSSION THREAD

**Discussion by fenestren** [indomus/forum#101] [HUB: Home Assistant]
Ciao a tutti,
non capisco come mai improvvisamente la connessione HTTPS non funziona più e, con essa, non funzionano più né la HA companion APP per iPhone né l'integrazione di Google Home Mini.
Ho DuckDNS e Caddy come Docke-Compose e uso Caddy come reverse proxy (ho seguito alla lettera tutte le guide sul sito).
Ho provato a ricreare da capo i container Caddy e DuckDNs e, seguendo la vostra guida, ho creato dapprima il Caddyfile:

```
When your config is ok comment block below to disable use of AMCE staging server
{
    acme_ca https://acme-staging-v02.api.letsencrypt.org/directory 
}

https://MIO_DOMINIO.duckdns.org { 
  respond "Saluti dal Crew inDomus.it"
  file_server
}
```

che mi dice che ho ottenuto correttamente i certificati e poi quello finale:

```
(https_header) {
  header {
    Strict-Transport-Security "max-age=31536000; includeSubdomains"
    X-XSS-Protection "1; mode=block"
    X-Content-Type-Options "nosniff"
    X-Frame-Options "SAMEORIGIN"
    Referrer-Policy "same-origin"
  }
}

https://MIO_DOMINIO.duckdns.org { 
  respond "Saluti dal Crew inDomus"
  file_server
}

https://MIO_DOMINIO.duckdns.org:443 { 
  import https_header
  reverse_proxy http://192.168.178.49:8123
}
```

dal quale però non ottengo il risultato sperato e tutto sembra bloccarsi.
Potreste aiutarmi, per favore?

Grazie mille!

**Comment by snakuzzo** [comment]
Troppi pochi dettagli. Solo con la config di caddy non possiamo dirti nulla.
Ricreare i container a cacchio non risolve i problemi.
Cosa logga HA quando provi a collegarti ?
Che errore/warning ti dà il browser ?

Ti inviterei a correggere il post usando correttamente il markdown. Grazie

**Reply by fenestren** [reply]
Ciao Diego,
attualmente riesco ad accedere ad HA solo dal PC che ospita Docker con i vari container tra cui appunto HA, Caddy e DuckDNS.
Se provo ad accedere tramite la APP per iPhone all'indirizzo locale (192.168.178.49:8123) dopo qualche tempo ottengo un errore di Timeout.
Il LOG di HA è quello nello screeshot allegato
![Schermata del 2021-04-06 19-36-13](https://user-images.githubusercontent.com/74239204/113754347-6b928480-970f-11eb-9926-70f839526b94.png)
Dato che neanche il Google Home Mini comunica più con HA, ho provato a rifare la procedura di test da Google Actions come descritto sulla guida, senza successo purtroppo.
Il log di DuckDNS a volte è ok (_Your IP was updated at Tue Apr 6 19:35:04 CEST 2021_) mentre altre riporta l'errore nell'update del IP (_Something went wrong, please check your settings Thu Gen 1 00:00:00 CET 1980_).
Il log di Caddy spesso riporta errori come questo: _"err_trace":"reverseproxy.statusError (reverseproxy.go:783)"_

Non saprei cosa altro fare.
Grazie del supporto.

**Reply by snakuzzo** [reply]
È ancora troppo poco...serve il log completo, non quello sintetico che vedi da gui.
Dove trovi i log clicca sul tastino in basso per farti mostrare il log completo.
Quanto al container duckdns, l'errore è tropo generico.
Per quanto riguarda caddy...stessa cosa.

**Reply by fenestren** [reply]
Ciao e grazie ancora! 
Ecco il log...non saprei cosa stia succedendo...potrebbe essere qualcosa legato alla configurazione di rete del mio PC host Ubuntu?
Prima ho anche avuto problemi di autorizzazione a fare il pull delle immagini dei container (mai successo).

```
ELIMINATO
```

**Reply by snakuzzo** [reply]
Mi hai preso troppo alla lettera.
Hai talmente tanti di quegli errori che è impossibile starti dietro.

Prendi quel log che hai postato qui e che ho cancellato e cerca di risolvere errore per errore partendo dall'alto
